### PR TITLE
updated read me

### PR DESCRIPTION
**Pull request by mvkvinod**

### CODE REVIEW

**Review by frenzydeets — CHANGES_REQUESTED** [review]
add more text please

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ### Hi there 👋
-This is for
+This is for test
```
Comment (frenzydeets) [inline README.md:2]: here

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ### Hi there 👋
+This is for test
```
Comment (frenzydeets) [inline README.md:3]: here

**Review by frenzydeets — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mvkvinod) [reply README.md:3]: ok

**Pull request merged**